chemprop/uncertainty/calibrator.py

```diff
@@ -68,20 +66,73 @@ def apply_calibration(self, preds, uncs) -> Tensor:
 
 @UncertaintyCalibratorRegistry.register("platt")
 class PlattCalibrator(UncertaintyCalibrator):
-    def calibrate(self, preds, uncs, targets, mask) -> Tensor:
-        ...
-        return
-
-    def apply_calibration(self, preds, uncs) -> Tensor:
-        ...
-        return
+    """A calibration method for classification datasets based on the Platt scaling algorithm, as
+    discussed in [guo2017]_ and [platt1999]_. In Platt's paper, he suggests using the number of
+    positive and negative examples in the dataset used to train the model to adjust the value of
+    target probabilities used to fit the parameters.
+
+    References
+    ----------
+    .. [guo2017] Guo, C.; Pleiss, G.; Sun, Y.; Weinberger, K. Q. "On calibration of modern neural
+        networks". ICML, 2017. https://arxiv.org/abs/1706.04599
+    .. [platt1999] Platt, J.. "Probabilistic Outputs for Support Vector Machines and Comparisons to
+        Regularized Likelihood Methods." Adv. Large Margin Classif. 1999, 10 (3), 61–74.
+    """
+
+    def calibrate(self, preds, targets, mask, training_targets: None | Tensor = None):
+        if torch.any((targets[mask] != 0) & (targets[mask] != 1)):
+            raise ValueError(
+                "Platt scaling is only implemented for binary classification tasks! Input tensor "
+                "must contain only 0's and 1's."
+            )
+
+        if training_targets is not None:
```

Comment:
Add a `logger.ingo()` with information like `Platt scaling for calibration uses Bayesian correction against training set overfitting, replacing calibration targets [0, 1] with adjusted values.` Similarly, in cases without `training_targets` provided, adding a sentence like `No Bayesian correction performed as part of class scaling.`